rules/go/shared/lang/dynamic_input.yml

```diff
@@ -26,6 +26,7 @@ auxiliary:
       - os.Args
       - os.Getenv()
       - func $<_>($<...>$<!>$<ARG> $<_>$<...>)
+      - func ($<...>) $<_>($<...>$<!>$<_> $<_>$<...>)$<...>
```

Comment:
Could you give an example of the use case for this pattern?

Reply:
https://github.com/Bearer/bearer-rules/pull/255/files#diff-8151f78949588df3c12a777162d4523b33ea3b8cd9b948d84dfe0a30b0dd8f6fR25